1018-modal-design-guidance.md

```diff
@@ -0,0 +1,29 @@
+---
+category: component design guidance
+state: draft
+created: 2024-10-17
+updated: 2024-10-22
+---
+
+# Modal Design Guidance
+
+[Modal](https://clarity.design/documentation/modal) component informs the users about a task and can contain information that requires immediate attention.
+
+## Guidance
+
+- **Must** use when you want to grab user's attention for an important message, such as errors that prevent a user from completing an action or actions that can't be reversed.
+- **Must** use to show additional information without losing the context of the parent page.
+- **Must** use to break down a complex workflow into a series of less complicated steps.
+- **Must** test responsiveness before picking the size of the modal. Different [Modal sizes](https://clarity.design/documentation/modal#sizes) are available in the pattern. Default being medium.
+- **Must not** use for non-essential information that is not related to the current workflow.
```

Comment:
```suggestion
- Design teams **must not** use modals for non-essential information that is unrelated to the current workflow, as this can distract users and disrupt their focus on important tasks.
```